definition_files.rst

```diff
@@ -87,23 +87,22 @@ Preferred bootstrap agents
 
 -  :ref:`docker <build-docker-module>` (images hosted on Docker Hub)
 -  :ref:`oras <build-oras>` (images from supporting OCI registries)
--  :ref:`library <build-library-module>` (images hosted on Library API Registries)
+-  :ref:`localimage <build-localimage>` (images saved on your machine)
 -  :ref:`scratch <scratch-agent>` (a flexible option for building a
    container from scratch)
 
 Other bootstrap agents
 ======================
 
+-  :ref:`library <build-library-module>` (images hosted on Library API Registries)
 -  :ref:`shub <build-shub>` (images hosted on Singularity Hub)
--  :ref:`localimage <build-localimage>` (images saved on your machine)
 -  :ref:`yum <build-yum>` (yum based systems such as CentOS and
    Scientific Linux)
 -  :ref:`debootstrap <build-debootstrap>` (apt based systems such as
    Debian and Ubuntu)
 -  :ref:`oci <cli-oci-bootstrap-agent>` (bundle compliant with OCI Image
```

Comment:
This one also doesnt have a section to link to

Reply:
I know. I had added the label at an artificial place, where the oci bundle was described in oci_runtime.rst, but that probably wasn't a good idea.  I took it out.